spec/CloudCode.spec.js

```diff
@@ -2784,6 +2784,26 @@ describe('afterFind hooks', () => {
     });
   });
 
+  it('should limit count query if limit is set', async done => {
+    const hook = {
+      method: function (req) {
+        expect(req.count).toBe(true);
+        return Promise.resolve();
+      },
+    };
+    spyOn(hook, 'method').and.callThrough();
+    Parse.Cloud.beforeFind('Stuff', hook.method);
```

Comment:
Why is spying on the hook relevant for this test?

Reply:
Don't know, I copied the previous test in the file and didn't know if it was needed :)